Makefile

```diff
@@ -319,7 +319,7 @@ $(foreach prog,$(PROGRAMS),$(eval $(call BUILD_ASSEMBLY,$(prog))))
 #
 default: core
 
-core: game platforms mods utility server
+core: dependencies game platforms mods utility server
```

Comment:
The technically correct option would be to set up the target dependencies properly, with `mods` depending on `game`, `game` depending on `dependencies`, and so on.

Our makefile is already a hack, so I don't really see the advantage of that over what @Phrohdoh has done here.  Ideally in the long run we will replace all of this with `core: dependencies <target that runs msbuild>`.